--- a/cmake/DD4hepBuild.cmake
+++ b/cmake/DD4hepBuild.cmake
@@ -705,8 +705,8 @@ macro(DD4HEP_SETUP_ROOT_TARGETS)
     dd4hep_debug("D++> Python version used for building ROOT ${ROOT_PYTHON_VERSION}" )
     if (NOT DD4HEP_RELAX_PYVER)
       dd4hep_debug("D++> Required python version ${REQUIRE_PYTHON_VERSION}")
-      FIND_PACKAGE(Python ${REQUIRE_PYTHON_VERSION} EXACT REQUIRED COMPONENTS Development)
-      FIND_PACKAGE(Python ${REQUIRE_PYTHON_VERSION} EXACT QUIET COMPONENTS Interpreter)
+      FIND_PACKAGE(Python ${REQUIRE_PYTHON_VERSION} REQUIRED COMPONENTS Development)
+      FIND_PACKAGE(Python ${REQUIRE_PYTHON_VERSION} QUIET COMPONENTS Interpreter)
     else()
       FIND_PACKAGE(Python REQUIRED COMPONENTS Development)
       FIND_PACKAGE(Python QUIET COMPONENTS Interpreter)
