diff --git a/extensions/handlers-exp/CMakeLists.txt b/extensions/handlers-exp/CMakeLists.txt
index 2cca4cf3..5ed6b5e2 100644
--- a/extensions/handlers-exp/CMakeLists.txt
+++ b/extensions/handlers-exp/CMakeLists.txt
@@ -10,8 +10,8 @@ set( na64exp_MODULE_SOURCES
         mk-calo-hit-by-time-cluster-id.cc
         dump-calo-hits.cc
         msadc-dft-epdf.cc
-        msadc-dump-selected.cc
-        sadc-dump-pulses.cc
+        #msadc-dump-selected.cc
+        #sadc-dump-pulses.cc
         msadc-fft-can-be-zero.cc
         msadc-fft-mean-amp.cc
         msadc-fft-plot-spectra.cc
@@ -30,7 +30,7 @@ set( na64exp_MODULE_SOURCES
         #msadc-wf-fit-peaks.cc  # xxx
         #msadc-p348reco-peak-append.cc  #?
         calo-hit-vs-evno.cc
-        sadc-subtract-mt.cc
+        #sadc-subtract-mt.cc
     )
 # collect headers recursively by wildcard; used only by install target
 file(GLOB_RECURSE na64exp_HEADERS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "*.hh" "*.h" "*.tcc" "*.hpp" )
