diff --git a/extensions/na64common/CMakeLists.txt b/extensions/na64common/CMakeLists.txt
index eaec2ecb..8604bf59 100644
--- a/extensions/na64common/CMakeLists.txt
+++ b/extensions/na64common/CMakeLists.txt
@@ -54,7 +54,7 @@ endif( NOT na64util_EXT_FOUND )
 
 # Look for ddd-ext
 find_path( dddext_INCLUDE_DIR
-           NAMES na64/ddd-ext/config.hh
+           NAMES na64/ddd-ext/config.h
            #PATH_SUFFIXES na64
            PATHS ${CMAKE_INSTALL_PREFIX}/include
                  ${dddext_PREFIX}/include
